integ-tests/src/commonTest/kotlin/dev/evo/elasticmagic/ExplanationTest.kt

```diff
@@ -0,0 +1,60 @@
+package dev.evo.elasticmagic
+
+import dev.evo.elasticmagic.doc.BoundField
+import dev.evo.elasticmagic.doc.RootFieldSet
+import dev.evo.elasticmagic.qf.FIXTURES
+import dev.evo.elasticmagic.qf.ItemDoc
+import dev.evo.elasticmagic.query.match
+import dev.evo.elasticmagic.types.TextType
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.shouldNotBe
+import kotlin.test.Test
+
+class ExplanationTest : ElasticsearchTestBase() {
+    override val indexName = "explanation-test"
```

Comment:
Let's call it merely `explanation`. The resulting index name will be `elasticmagic-tests_explanation`